src/sites/amsterdam/components/shared/amsterdam.css

```diff
@@ -0,0 +1,12 @@
+.amsterdam-form-navigation {
```

Comment:
Please update the prefix to `ams`

Reply:
Moved to same folder as `FormNavigation`